worker/worker.go

```diff
@@ -160,6 +160,9 @@ type Bus interface {
 	Accounts(ctx context.Context) ([]api.Account, error)
 	UpdateSlab(ctx context.Context, s object.Slab, contractSet string, goodContracts map[types.PublicKey]types.FileContractID) error
 
+	AddUploadedSector(ctx context.Context, uID api.UploadID, id types.FileContractID, root types.Hash256) error
```

Comment:
I considered `TrackUploadedSector`, might be slightly better?

Reply:
Maybe `TrackUpload` and `FinishUpload` for starting and finishing them as well as `AddUploadingSector` for adding a sector to the upload?

Reply:
Added it, don't really see why an upload can't be lazily initiated and we have the explicit `TrackUpload` one but maybe it'll pay off in the future if we track other upload-related info